core/src/Kokkos_Graph.hpp

```diff
@@ -91,10 +91,12 @@ struct [[nodiscard]] Graph {
     (*m_impl_ptr).instantiate();
   }
 
-  void submit() const {
+  void submit(const execution_space& exec) const {
     KOKKOS_EXPECTS(bool(m_impl_ptr))
-    (*m_impl_ptr).submit();
+    (*m_impl_ptr).submit(exec);
   }
+
+  void submit() const { submit(get_execution_space()); }
```

Comment:
Any reason to prefer a new overload over a default argument?

Reply:
None for now... I'll change that :wink:

Reply:
So, I wanted to write
```c++
  void submit(const execution_space& exec = get_execution_space()) const;
```
but as I expected, this isn't allowed, because
```bash
error: a nonstatic member reference must be relative to a specific object
```

Reply:
So we'll need both overloads if we want to continue supporting `submit()`.